README.md

```diff
@@ -14,30 +28,99 @@ To install multiple versions of Node.js, you may find it easier to use a node ve
 * [nvm](https://github.com/creationix/nvm)
 * [n](https://github.com/tj/n)
 
-### Installation
+
+## Installation
+
+Clone this repo and install its dependencies...
 
 ```
 $ npm install
 ```
 
-### Usage
+You'll then need to duplicate the `config.sample.json` file as `config.json` and update its contents...
```

Comment:
Who do you get the contents from? how about

Please rename the `config.sample.json` file to `config.json`.  _(who)_  can provide you with the values to update `config.json` with.